CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -0,0 +1,81 @@
+- Add instructions to install Antrea in `encap` mode in AKS. ([#5901](https://github.com/antrea-io/antrea/pull/5901), [@antoninbas])
+- Add Flexible IPAM design details in `antrea-ipam.md`. ([#5339](https://github.com/antrea-io/antrea/pull/5339), [@gran-vmv])
+
+### Fixed
+
+- Store NetworkPolicy in filesystem as fallback data source to let antre-agent fallback to use the files if it can't connect to antrea-controller on startup. ([#5739](https://github.com/antrea-io/antrea/pull/5739), [@tnqn])
+- Enable Pod network after realizing initial NetworkPolicies to avoid traffic from/to Pods bypassing NetworkPolicy when antrea-agent restarts. ([#5777](https://github.com/antrea-io/antrea/pull/5777), [@tnqn])
+- Fix Clean-AntreaNetwork.ps1 invocation in Prepare-AntreaAgent.ps1 for containerized OVS on Windows. ([#5859](https://github.com/antrea-io/antrea/pull/5859), [@antoninbas])
+- Add missing space to kubelet args in Prepare-Node.ps1 so that kubelet can start successfully on Windows. ([#5858](https://github.com/antrea-io/antrea/pull/5858), [@antoninbas])
+- Fix `antctl trace-packet` command failure which is caused by arguments missing issue. ([#5838](https://github.com/antrea-io/antrea/pull/5838), [@luolanzone])
+- Support Local ExternalTrafficPolicy for Services with ExternalIPs when Antrea proxyAll mode is enabled. ([#5795](https://github.com/antrea-io/antrea/pull/5795), [@tnqn])
+- Set `net.ipv4.conf.antrea-gw0.arp_announce` to 1 to fix an ARP request leak when a Node or hostNetwork Pod accesses a local Pod and AntreaIPAM is enabled. ([#5657](https://github.com/antrea-io/antrea/pull/5657), [@gran-vmv])
+- Skip enforcement of ingress NetworkPolicies rules for hairpinned Service traffic (Pod accessing itself via a Service). ([#5687](https://github.com/antrea-io/antrea/pull/5687) [#5705](https://github.com/antrea-io/antrea/pull/5705), [@GraysonWu])
+- Add host-local IPAM GC on startup to avoid potential IP leak issue after antrea-agent restart. ([#5660](https://github.com/antrea-io/antrea/pull/5660), [@antoninbas])
+- Fix antrea-ubi image build failure with ubi8 image. ([#5723](https://github.com/antrea-io/antrea/pull/5723), [@antoninbas])
+- Remove redundant log in `fillPodInfo`/`fillServiceInfo` to fix log flood issue, and update `DestinationServiceAddress` for deny connections. ([#5592](https://github.com/antrea-io/antrea/pull/5592) [#5704](https://github.com/antrea-io/antrea/pull/5704), [@yuntanghsu])
+- Enhance HNS network initialization on Windows to avoid some corner cases. ([#5841](https://github.com/antrea-io/antrea/pull/5841), [@XinShuYang])
+- Fix endpoint querier rule index in response to improve troubleshooting. ([#5783](https://github.com/antrea-io/antrea/pull/5783), [@qiyueyao])
+- Enhance ExternalIPPool validation to avoid unexpected behaviors. ([#5898](https://github.com/antrea-io/antrea/pull/5898), [@aroradaman])
```

Comment:
As it hasn't been merged, we should remove it if there is no update today.

Reply:
Will remove this later if there is still no change for the PR.

Reply:
Removed this considering it's not critical and might not be merged in time.